_layouts/base.html

```diff
@@ -10,6 +10,41 @@
         {% if page.description %}<meta name="description" content="{{ page.description }}">{% endif %}
         <link href="{{ site.baseurl }}/css/custom.css" rel="stylesheet" type="text/css">
         <link rel="stylesheet" href="https://cdn.rawgit.com/jpswalsh/academicons/master/css/academicons.min.css">
+        <!--Below is the schema annotation for blogposts-->
+        {% if page.layout == 'post' %}
+        <script type="application/ld+json">
+          {
+            "@context": "https://schema.org",
+            "@type": "BlogPosting",
+            "mainEntityOfPage": {
+              "@type": "WebPage",
+              "@id": "{{ site.url }}/posts/"
+              },
+            "headline": "{{ page.title }}",
+            "description": "{{ page.excerpt | strip_html | strip_newlines | truncate: 200 }}",
+            "image": "{{ page.image }}",
+            {% for author in page.authors %}
+            "author": {
+                  "@type": "Person",
+                  "name": "{{ author }}",
```

Comment:
The name here will not be the full name but the GitHub id. You need to retrieve first and last name from the `site.data.people`, sth like `{{ site.data.people[author].first-name }} {{ site.data.people[author].last-name }}

Reply:
True. 💯 
Thanks for the tip! 🙌